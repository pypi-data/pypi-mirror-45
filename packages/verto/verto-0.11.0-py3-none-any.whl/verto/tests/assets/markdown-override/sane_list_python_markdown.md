A Paragraph.
* Not a list item.

1. Ordered list item.
* Not a separate list item.
