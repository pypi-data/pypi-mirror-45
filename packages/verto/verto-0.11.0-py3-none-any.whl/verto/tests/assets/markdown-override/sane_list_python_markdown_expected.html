<p>A Paragraph.
* Not a list item.</p>
<ol>
<li>Ordered list item.
* Not a separate list item.</li>
</ol>
